### PR TITLE
Fix gke provisioning

### PR DESCRIPTION
**Pull request by vponomaryov**
Currently, the GKE provisioning is broken in 2 places.

1) `GKE monitoring node label setting`

If we run a test on GKE backend then we get following error creating monitoring node:
    
```
sdcm.cluster.NodeError: Timeout while running 'ex_set_node_labels' method on GCE instance '3260019724311279728'
```
    
So, fix it by adding the required 'monitorid' label to the 'tags' attribute that will be added
by the common code and remove the failing part from the GKE's monitoring class.

2) `GKE cluster creation`

For the moment, if we try to create a GKE cluster we get following error:
    
```
ERROR: (gcloud.container.clusters.create) ResponseError:  code=403, message=Request had insufficient authentication scopes.
```
    
The root cause for it is absence of enabled `cloud-platform` scope for the SCT runner isntance.
So, fix it by specifying this scope creating GCE instance.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/sct_runner.py [hunk 1]
```diff
@@ -667,6 +667,11 @@ def _create_instance(self,
                 "block-project-ssh-keys": "true",
                 "ssh-keys": f"{self.LOGIN_USER}:{self.key_pair.public_key.decode()}",
             },
+            # NOTE: following is needed for the possibility to use the K8S/GKE APIs
+            ex_service_accounts=[{
```
Comment (fruch) [inline sdcm/sct_runner.py:671]: Do we understand how it was working in the old project ? The default service had it enabled somehow ?

File: sdcm/cluster_k8s/gke.py [hunk 2]
```diff
@@ -478,10 +479,3 @@ def sort_by_index(node):
 
         instances = sorted(instances, key=sort_by_index)
         return instances
-
-    # pylint: disable=protected-access
-    def _create_node(self, instance, node_index, dc_idx):
-        node = super()._create_node(instance=instance, node_index=node_index, dc_idx=dc_idx)
-        node._instance_wait_safe(
```
Comment (fruch) [inline sdcm/cluster_k8s/gke.py:485]: So this code path we added as part of the multi tenant support, and didn't tested it ?

**Review by fruch — APPROVED** [review]
LGTM

File: sdcm/cluster_k8s/gke.py (hunk 2 above)
Comment (vponomaryov) [reply sdcm/cluster_k8s/gke.py:485]: It worked at that time. Failures labels setting started appearing last Friday and all the time after it. So, I guess something has changed on the GKE server side, update or something else.
So, I just started using another solution that works "now".

Comment (fruch) [reply sdcm/cluster_k8s/gke.py:485]: o.k.

**Pull request merged**